shaka/shaka/ColorExtension.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ColorExtension.swift
+//  shaka
+//
+//  Created by 유정인 on 2022/07/29.
+//
+
+import UIKit
+
+extension UIColor {
+    static let cellBackground1 = UIColor(named: "CellBackground1")
+    static let cellBackground2 = UIColor(named: "CellBackground2")
+    static let cellBackground3 = UIColor(named: "CellBackground3")
+    static let cellBackground4 = UIColor(named: "CellBackground4")
```

Comment:
1, 2, 3, 4 보다는 버튼 색상을 이름에 맞게 설정하면 어떤 색상인지 보기 편할 것 같아요
```suggestion
    static let cellBackgroundBlue = UIColor(named: "CellBackground1")
    static let cellBackgroundSky = UIColor(named: "CellBackground2")
    // 등등
```

Reply:
수정하겠습니다~